### ISSUE TITLE
IE8 Null Object Error

### ISSUE THREAD

**Issue by longoria** [doctyper/flexie#2]
Thanks for the awesome library!

IE8 on Windows XP reports the following error with 0.4 and the same CSS in Issue 1

Line: 402
Error: 'property' is null or not an object

**Comment by doctyper** [comment]
It would help if you included the markup you're currently using, maybe as a Gist? Because this issue stems from a CSS framework, it could be any combination of classes that are causing the error.

**Comment by longoria** [comment]
HTML:
https://gist.github.com/662665

CSS:
https://gist.github.com/662674

**Comment by doctyper** [comment]
I've just landed a fix for this. Nutshell: I wasn't accounting for elements having multiple classes, and thus multiple flexbox rules. This is now resolved.

One thing though, Flexie will not work with inline-styles. It must overwrite these styles to add its own positioning values. So as is, your sample will not work. But moving the inline styles to a stylesheet should get your example working:

HTML:
https://gist.github.com/663197

CSS:
https://gist.github.com/663187
